# HG changeset patch
# User Rik <rik@octave.org>
# Date 1437235011 25200
#      Sat Jul 18 08:56:51 2015 -0700
# Node ID 2f9119bb3fe5b73b12e873b4dec1c3e0d851ee87
# Parent  6b7c10920dfeb922d0e5c4856cbd78067275cd25
build: Add EXTRA_DIST files frob libgui/ and test/ to tarball.

* libgui/module.mk, test/module.mk: Add local EXTRA_DIST file lists to master
EXTRA_DIST variable so that they are included in tarball.

diff --git a/libgui/module.mk b/libgui/module.mk
--- a/libgui/module.mk
+++ b/libgui/module.mk
@@ -111,16 +111,18 @@ am__v_lrelease_1 =
 
 DIRSTAMP_FILES += \
   libgui/languages/$(octave_dirstamp)
 
 libgui_EXTRA_DIST += \
   $(TRANSLATIONS) \
   libgui/default-qt-settings.in
 
+EXTRA_DIST += $(libgui_EXTRA_DIST)
+
 libgui_DISTCLEANFILES += \
   libgui/default-qt-settings \
   $(LOCALES)
 
 CLEANFILES += $(libgui_CLEANFILES)
 DISTCLEANFILES += $(libgui_DISTCLEANFILES)
 MAINTAINERCLEANFILES += $(libgui_MAINTAINERCLEANFILES)
 
diff --git a/test/module.mk b/test/module.mk
--- a/test/module.mk
+++ b/test/module.mk
@@ -98,16 +98,18 @@ BUILT_SOURCES += $(GENERATED_TEST_FILES)
 
 test_EXTRA_DIST += \
   test/build-sparse-tests.sh \
   test/build-bc-overload-tests.sh \
   test/bc-overloads-expected \
   test/build_bc_overloads_expected.m \
   $(TEST_FILES)
 
+EXTRA_DIST += $(test_EXTRA_DIST)
+
 test_CLEANFILES += \
   $(GENERATED_BC_OVERLOADS_FILES) \
   $(GENERATED_TEST_FILES)
 
 test_DISTCLEANFILES += \
   test/.gdbinit \
   test/fntests.log
 
